Source/options.html

```diff
@@ -26,6 +26,14 @@
             To improve the accuracy of the estimation, <br />
             please fill in below how much CO<sub>2</sub> your car emits (in g/km): <br />
```

Comment:
this line needs to be changed too. I suggest:

"... please fill in below how much fuel your car consumes or how much CO<sub>2</sub> it emits: "